### ISSUE TITLE
Organic group sync

### ISSUE THREAD

**Issue by laurenzfg** [studieren-ohne-grenzen/vogelnest#31]
Before the shutdown of the legacy dashboard, we need to migrate the functionality of this repo (https://github.com/studieren-ohne-grenzen/sog_og_ldap) to the new vogelnest. I don't really understand why we need to sync LDAP groups to Drupal, perhaps to grant CiviCRM access permissions? Any ideas?

**Comment by marcelklehr** [comment]
I don't think we currently use this drupal module. I can check later, though.
